openfpga_flow/tasks/skywater_openfpga_task

```diff
@@ -0,0 +1 @@
+/home/apond/sofa/SCRIPT/skywater_openfpga_task
```

Comment:
Can you remove this symbolic link?

Reply:
Yep, I must have merged the other branch into this one before I deleted the symlink.